### ISSUE TITLE
Invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by wd021** [Layr-Labs/eigenlayer-cli#78]
**Describe the bug**
When trying to register an operator, hit this runtime error. Rerunning it worked but nevertheless, posting it here incase its a new bug.

**To Reproduce**
Register an operater using eigenlayer cli

```
2024-03-12T10:21:42.942Z        DEBUG   logging/zap_logger.go:45        Sending transaction
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x80634c]

goroutine 1 [running]:
github.com/ethereum/go-ethereum/core/types.(*Transaction).Hash(0x0?)
        /home/danny/go/pkg/mod/github.com/ethereum/go-ethereum@v1.13.12/core/types/transaction.go:464 +0x2c
github.com/Layr-Labs/eigensdk-go/chainio/txmgr.(*SimpleTxManager).Send(0xc0003247d0, {0xd05158, 0x11e94a0}, 0x0?)
        /home/danny/go/pkg/mod/github.com/!layr-!labs/eigensdk-go@v0.1.1/chainio/txmgr/txmgr.go:123 +0x66a
github.com/Layr-Labs/eigensdk-go/chainio/clients/elcontracts.(*ELChainWriter).RegisterAsOperator(0xc00031e000, {0xd05158, 0x11e94a0}, {{0xc00014a420, 0x2a}, {0xc00014a450, 0x2a}, {0xc00014a480, 0x2a}, 0x0, ...})
        /home/danny/go/pkg/mod/github.com/!layr-!labs/eigensdk-go@v0.1.1/chainio/clients/elcontracts/writer.go:129 +0x1db
github.com/Layr-Labs/eigenlayer-cli/pkg.OperatorCmd.RegisterCmd.func1(0xc0002bf8b8?)
        /home/danny/eigenlayer-cli/pkg/operator/register.go:126 +0x89c
github.com/urfave/cli/v2.(*Command).Run(0xc0001589a0, 0xc000130300, {0xc000124c40, 0x2, 0x2})
        /home/danny/go/pkg/mod/github.com/urfave/cli/v2@v2.27.1/command.go:279 +0x9dd
github.com/urfave/cli/v2.(*Command).Run(0xc000158dc0, 0xc0001301c0, {0xc00011c2d0, 0x3, 0x3})
        /home/danny/go/pkg/mod/github.com/urfave/cli/v2@v2.27.1/command.go:272 +0xc2e
github.com/urfave/cli/v2.(*Command).Run(0xc000159080, 0xc000130080, {0xc000036080, 0x4, 0x4})
        /home/danny/go/pkg/mod/github.com/urfave/cli/v2@v2.27.1/command.go:272 +0xc2e
github.com/urfave/cli/v2.(*App).RunContext(0xc000156000, {0xd05158?, 0x11e94a0}, {0xc000036080, 0x4, 0x4})
        /home/danny/go/pkg/mod/github.com/urfave/cli/v2@v2.27.1/app.go:337 +0x5db
github.com/urfave/cli/v2.(*App).Run(...)
        /home/danny/go/pkg/mod/github.com/urfave/cli/v2@v2.27.1/app.go:311
main.main()
        /home/danny/eigenlayer-cli/cmd/eigenlayer/main.go:34 +0x218
```

**Screenshots**
![Screenshot 2024-03-12 at 10 23 10](https://github.com/Layr-Labs/eigenlayer-cli/assets/2752586/592a5acf-f94d-42f2-94cc-4055e71d6f72)

**OS details**
Ubuntu 20

**Comment by shrimalmadhur** [comment]
@wd021 found the issue - `v0.1.2` of eigensdk will fix this - https://github.com/Layr-Labs/eigensdk-go/releases/tag/v0.1.2 - I will close it once it's fixed

**Comment by wd021** [comment]
awesome 👍

**Comment by TweedBeetle** [comment]
Could someone explain how to fix this without waiting for v0.1.2 of eigensdk?  

Simply running `go get github.com/Layr-Labs/eigensdk-go` does not seem to work.

**Comment by shrimalmadhur** [comment]
this got fixed as we updated eigensdk.
